public/js/pimcore/object/tags/table.js

```diff
@@ -225,16 +225,7 @@ pimcore.object.tags.table = Class.create(pimcore.object.tags.abstract, {
             tbar: tbar,
             viewConfig: {
                 markDirty: false,
-                forceFit: true,
-                listeners: {
-                    refresh: function (dataview) {
-                        Ext.suspendLayouts();
-                        Ext.each(dataview.panel.columns, function (column) {
-                            column.autoSize();
-                        });
-                        Ext.resumeLayouts(true);
-                    }
-                }
+                forceFit: true
```

Comment:
```suggestion
                forceFit: true,
                listeners: {
                    refresh: function (dataview) {
                        Ext.suspendLayouts();
                        let totalWidth = 0;
                        const columns = dataview.panel.columns;
                        
                        Ext.each(columns, function (column) {
                            totalWidth += column.getWidth();
                        });
                        
                        if (totalWidth > this.fieldConfig.width){
                            Ext.each(columns, function (column) {
                                column.autoSize();
                            });
                        }

                        Ext.resumeLayouts(true);
                    }.bind(this)
                }
```
This would be the best of the 2 world, so when the column

Reply:
Without listener (as the change in this PR)
<img width="578" height="307" alt="image" src="https://github.com/user-attachments/assets/1dcf6170-f285-4f84-9d96-507d0bb3e311" />

With listener (https://github.com/pimcore/admin-ui-classic-bundle/pull/566)
<img width="578" height="307" alt="image" src="https://github.com/user-attachments/assets/f8c988c1-2b6b-4e0d-90eb-93e35040235a" />


with the suggestion above
<img width="578" height="307" alt="image" src="https://github.com/user-attachments/assets/89e8895f-513f-4802-b607-1dd4b79b10bb" />